src/launcher.ts

```diff
@@ -0,0 +1,126 @@
+import { promisify } from 'util'
+import { performance, PerformanceObserver } from 'perf_hooks'
+
+import * as BrowserstackLocalLauncher from 'browserstack-local'
+import logger from '@wdio/logger'
+import type { Capabilities, Services, Options } from '@wdio/types'
+
+import { BrowserstackConfig } from './types'
+
+const log = logger('@browserstack/wdio-browserstack-service')
+
+type BrowserstackLocal = BrowserstackLocalLauncher.Local & {
+    pid?: number;
+    stop(callback: (err?: any) => void): void;
+}
+
+export default class BrowserstackLauncherService implements Services.ServiceInstance {
```

Comment:
Can we rename BrowserstackLauncherService to BrowserstackLocalService? Or add comments to intend the functionality of this file?

Reply:
Renaming would require changes in multiple other places as well, didn't change the naming convention already being used by the existing service. Will try to add some comments about the changes made.